requests_kerberos/kerberos_.py

```diff
@@ -71,6 +90,9 @@ def _negotiate_value(response):
         regex = re.compile('(?:.*,)*\s*Negotiate\s*([^,]*),?', re.I)
         _negotiate_value.regex = regex
 
+    if response is None: # we're doing an initial request- just use an empty negotiate value
```

Comment:
Really minor style nit, but this comment is long enough that I'd rather see it in a block above the `if` than as an in-line comment.